contracts/exchange/api/BaseContract.sol

```diff
@@ -406,4 +416,68 @@ contract BaseContract is AccessControlUpgradeable, ReentrancyGuardUpgradeable {
         L2ContractHelper.computeCreate2Address(address(this), salt, getDepositProxyBytecodeHash(), constructorInputHash)
       );
   }
+
+  function _getTotalAccountValueUSDT(Account storage account) internal view returns (BI memory) {
+    uint dec = _getBalanceDecimal(Currency.USDT);
+
+    BI memory totalValue = _getBalanceValueInQuoteCurrencyBI(account.spotBalances, Currency.USDT);
+
+    for (uint256 i; i < account.subAccounts.length; ++i) {
+      SubAccount storage subAcc = _requireSubAccount(account.subAccounts[i]);
+      BI memory subValueInQuote = _getSubAccountValueInQuote(subAcc);
+      BI memory subValueInUSDT = _convertCurrency(subValueInQuote, subAcc.quoteCurrency, Currency.USDT);
+
+      totalValue = totalValue.add(subValueInUSDT);
+    }
+
+    return totalValue;
+  }
+
+  function _getBalanceValueInQuoteCurrencyBI(
+    mapping(Currency => int64) storage balances,
+    Currency quoteCurrency
+  ) internal view returns (BI memory) {
+    uint64 dec = _getBalanceDecimal(quoteCurrency);
+    BI memory total = BI(0, dec);
```

Comment:
nit: zero can always be delcared as BI(0,0)

Reply:
Updated!